.storybook/stories/spinner/spinner.stories.ts

```diff
@@ -46,19 +31,114 @@ const defaultParameters: Parameters = {
-  { clrInline: false, clrInverse: true, clrMedium: false, clrSmall: true },
-  // inline spinners
-  { clrInline: true, clrInverse: false },
-  { clrInline: true, clrInverse: true },
-];
-
-setupStorybook(ClrSpinnerModule, defaultStory, defaultParameters, variants);
+const SpinnerTemplate: Story = args => ({
+  template: `
+  <div style="text-align: center">
+    <clr-spinner
+      [clrInverse]="clrInverse"
+      [clrSmall]="clrSmall"
+      [clrMedium]="clrMedium"
+      [clrInline]="clrInline"
+    >
+      {{text}}
+    </clr-spinner>
+    <br *ngIf="!clrInline" />
+    {{text}}
+  </div>
+  `,
+  props: args,
+});
+
+const SpinnerAllTemplate: Story = args => ({
+  template: `
+  <div style="text-align: center">
+    <h5>Large Spinners</h5>
+    
+    <h6>Default</h6>
+    <clr-spinner> {{text}} </clr-spinner>
+    <h6>Inverse</h6>
+    <clr-spinner [clrInverse]="true"> {{text}} </clr-spinner>
+  
+    <h5>Medium Spinners</h5>
+
+    <h6>Default</h6>
+    <div style="margin-top:5px">
+      <clr-spinner [clrMedium]="true"> {{text}} </clr-spinner>
+    </div>
+    <h6>Inverse</h6>
+    <div style="margin-top:5px">
+      <clr-spinner [clrMedium]="true" [clrInverse]="true"> {{text}} </clr-spinner>
+    </div> 
+
+    <h5>Small Spinners</h5>
+
+    <h6>Default</h6>
+    <div style="margin-top:5px">
+      <clr-spinner [clrSmall]="true"> {{text}} </clr-spinner>
+    </div>
+    <h6>Inverse</h6>
+    <div style="margin-top:5px">
+      <clr-spinner [clrSmall]="true" [clrInverse]="true"> {{text}} </clr-spinner>
+    </div>
+
+    <h5>Inline Spinners</h5>
+
+    <h6>Default</h6>
+    <div style="margin-top:5px">
+      <clr-spinner [clrInline]="true"></clr-spinner> {{text}}
+    </div>
+    <h6>Inverse</h6>
+    <div style="margin-top:5px">
+      <clr-spinner [clrInline]="true" [clrInverse]="true"></clr-spinner> {{text}}
+    </div>
+  </div>
+  `,
+  props: args,
+});
+
+export const Spinner: StoryObj = {
+  render: SpinnerTemplate,
+};
+
+export const SpinnerInverse: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrInverse: true },
+};
+
+export const SpinnerMedium: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrMedium: true },
+};
+
+export const SpinnerMediumInverse: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrMedium: true, clrInverse: true },
+};
+
+export const SpinnerSmall: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrSmall: true },
+};
+
+export const SpinnerSmallInverse: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrSmall: true, clrInverse: true },
+};
+
+export const SpinnerInline: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrInline: true },
+};
+
+export const SpinnerInlineInverse: StoryObj = {
+  render: SpinnerTemplate,
+  args: { clrInline: true, clrInverse: true },
+};
+
+export const SpinnerShowcase: StoryObj = {
+  render: SpinnerAllTemplate,
+  parameters: {
+    actions: { disable: true },
+    controls: { disable: true },
+  },
+};
```

Comment:
You can leave these without the spinner word because they are already under the Spinner section and that's enough. The only difference is for the 'Default' one which currently we can't name with that because of the condition on visual tests.

```suggestion
export const Inverse: StoryObj = {
  render: SpinnerTemplate,
  args: { clrInverse: true },
};

export const Medium: StoryObj = {
  render: SpinnerTemplate,
  args: { clrMedium: true },
};

export const MediumInverse: StoryObj = {
  render: SpinnerTemplate,
  args: { clrMedium: true, clrInverse: true },
};

export const Small: StoryObj = {
  render: SpinnerTemplate,
  args: { clrSmall: true },
};

export const SmallInverse: StoryObj = {
  render: SpinnerTemplate,
  args: { clrSmall: true, clrInverse: true },
};

export const Inline: StoryObj = {
  render: SpinnerTemplate,
  args: { clrInline: true },
};

export const InlineInverse: StoryObj = {
  render: SpinnerTemplate,
  args: { clrInline: true, clrInverse: true },
};

export const Showcase: StoryObj = {
  render: SpinnerAllTemplate,
  parameters: {
    actions: { disable: true },
    controls: { disable: true },
  },
};
```